### PR TITLE
TOC Sync: Move files under Get Started.

### LINKED ISSUE #8711: [Docs] Create top-level Get started section
Move the following files under Getting started:
./sources/getting-started/_index.md	
./sources/fundamentals/_index.md	
./sources/fundamentals/architecture/_index.md	
./sources/fundamentals/architecture/components/_index.md	
./sources/fundamentals/architecture/deployment-modes/_index.md	
./sources/fundamentals/architecture/rings/_index.md	
./sources/fundamentals/labels.md	
./sources/fundamentals/overview/_index.md

### PR DESCRIPTION
**Pull request by JStickler**
**Which issue(s) this PR fixes**:
Fixes #8711 

**Special notes for your reviewer**:
Part of the Database products table of contents alignment project.
Moves Getting Started and Fundamentals files under Get Started in TOC.
Updates front matter for the topic (headings, page weights, descriptions, aliases to redirect from old URL to new URL)
Fixes links broken by the move.

### CODE REVIEW

**Review by slim-bean — APPROVED** [review]
LGTM!

**Review by jdbaldry — COMMENTED** [review]
All the aliases should be changed to be relative rather than absolute paths otherwise old versions of documentation can steal redirects from latest.

For example, if in the future `docs/sources/get-started/_index.md` is removed. The indexed page `/docs/loki/latest/get-started/` will be redirected to the last version that had that alias defined.

File: docs/sources/get-started/labels.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+title: Understanding labels
+description: Explains how to Loki uses labels to define log streams.
+weight: 600
 aliases:
     - /docs/loki/latest/getting-started/labels/
+    - /docs/loki/latest/fundamentals/labels/
+    - /docs/loki/latest/get-started/labels/
 ---
-# Labels
+# Understanding labels
```
Comment (jdbaldry) [inline docs/sources/get-started/labels.md:11]: We try to avoid gerunds so I believe "Understand labels" would be prefered.

**Comment by JStickler** [comment]
Chatted with @jdbaldry, OK to merge this PR and do a second PR to update the alias formats.

**Pull request merged**